src/main.cpp

```diff
@@ -2239,16 +2239,23 @@ CAmount GetBlockSubsidy(int nHeight, const Consensus::Params& consensusParams)
     }
 }
 
+static std::atomic<bool> IBDLatchToFalse{false};
+// testing-only, allow initial block down state to be set or reset
+bool TestSetIBD(bool ibd) {
```

Comment:
This returns the old state in case that might be useful in the future (you could, for example, during a test, ensure that IBD is disabled, run some test code, then restore it to its original setting), but this functionality isn't currently used.